src/Hedera/Protobuf/basic_types.proto

```diff
@@ -0,0 +1,1594 @@
+syntax = "proto3";
```

Comment:
This file has 1600 lines, do we use everything?

Reply:
It's better for maintainance when we need to udpate the proto tbh